airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4313,6 +4338,12 @@ static unsigned long sendDataToOptionalApis(const String &data) {
 		send_csv(data);
 	}
 
+	if (cfg::send2sd)
+	{
+		debug_outln_info(F("## Logging to SD: "));
+		send_csv(data);
```

Comment:
@Warukira do we need this for anything other than debugging?

Reply:
@chegejames , now we do not need it for anything else.